front/src/applications/editor/tools/rangeEdition/components/RangeEditionLeftPanel.tsx

```diff
@@ -53,6 +72,66 @@ const RangeEditionLeftPanel = () => {
     });
   };
 
+  const unselectSwitch = (sw: string) => () => {
```

Comment:
Rename sw => swId or switchId
```suggestion
  const unselectSwitch = (swId: string) => () => {
```

Reply:
done